packages/graphql/src/services/sharing.ts

```
@@ -3299,6 +3368,335 @@ export async function transferLiveQuizOwnership(
+    )
+    await recomputeDerivedPermissions(
+      { practiceQuizId: id, userId: ctx.user.sub, updateAccessRequests: false },
+      prisma
+    )
+
+    return updated
+  })
+
+  // return info for new admin permission and corresponding cache update
+  const permission = updatedPracticeQuiz.directPermissions[0]
+  return permission && permission.user
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)